### DISCUSSION TITLE
Hooks not fired on relation field changes

### DISCUSSION THREAD

**Discussion by molotow11** [keystonejs/keystone#9052] [Questions]
Hi, Guys!

I have the following relation:

User (single) -> (many) Article

When I am changing User's aritcles assignment, this is not fires Article hooks. 

I am expecting that reference field update will fire the hooks and I can update some data based on this.
Please let me know if this possible to fix.

**Comment by dcousens** [comment]
@molotow11 what `operation` would you expect to fire for an `Article`?

**Comment by molotow11** [comment]
I expect that `update` operation should fire, because Article's `userId` field is actually updated
